validator_instance.go

```diff
@@ -190,7 +190,7 @@ func (v *Validate) ValidateMap(data map[string]interface{}, rules map[string]int
 	return v.ValidateMapCtx(context.Background(), data, rules)
 }
 
-// RegisterTagNameFunc registers a function to get alternate names for StructFields.
+// RegisterTagNameFunc registers a function to get alternate names (other than sturct field names) from FieldName.
```

Comment:
I don't think this is 100% correct either as if does not get them from `FieldName` but the `reflect.StructField`; perhaps it's still not specific enough?